x/liquidvesting/keeper/send_restriction_test.go

```diff
@@ -81,9 +81,8 @@ func (suite *KeeperTestSuite) TestKeeper_BankSend() {
 				lockedDenom, err := types.GetLockedRepresentationDenom("stake")
 				suite.Require().NoError(err)
 
-				pool, found, err := suite.pk.GetPool(ctx, 1)
+				pool, err := suite.pk.GetPool(ctx, 1)
 				suite.Require().NoError(err)
-				suite.Require().True(found)
```

Comment:
Can you please add back these tests by changing them to make sure the returned error is `ErrNotFound`? We might need to create a new `testutils` method to do this

Reply:
`err == nil` means the pool was found, so we don't need further tests after `suite.Require().NoError(err)`. I added `suite.Require().ErrorIs(err, collections.ErrNotFound)` to where we originally had `suite.Require().False(found)` tests.